10-habits/5_integration/Makefile

```diff
@@ -18,3 +18,6 @@ lint:
 
 generate:
 	@protoc -I=api/proto/ --go_out=api/ --go_opt=paths=source_relative --go-grpc_out=api/ --go-grpc_opt=paths=source_relative api/proto/*.proto
```

Comment:
Sidenote: I think this should be moved to a `//go:generate `, I think this is more common for smaller Go projects. For bigger projects buf.dev is now an option.